compiler/src/dmd/lexer.d

```diff
@@ -3072,6 +3088,66 @@ class Lexer
         return u;
     }
 
+    version (DMDLIB)
+    {
+        /***************************************************
+     * Parse a comment embedded between t.ptr and p.
+     * Remove trailing blanks and tabs from lines.
+     * Replace all newlines with \n.
+     * Preserve leading comment string in each line.
+     * Append to previous one for this token.
+     */
+        private void getRawComment(Token* t) pure
```

Comment:
Isn't intended properly, also you don't need that many stars

Reply:
Oops, missed this. Will fix.